layouts/shortcodes/testimonials.html

```diff
@@ -0,0 +1,26 @@
+{{ $data := index site.Data site.Language.Lang }}
+
+{{ if $data.testimonials.testimonials.enable }}
+{{ with $data.testimonials.testimonials }}
+
+<section class="max-w-7xl mx-auto my-12">
```

Comment:
The style hasn't been changed much from how the testimonials currently appear on the services page. I know that will change, so I'll create a follow on story to make those updates.